packages/teams-js/src/public/meeting.ts

```diff
@@ -518,4 +518,41 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
     registerHandler('meeting.meetingReactionReceived', handler);
   }
+  /**
+   * Nested namespace for functions to control behavior of the app share button
+   *
+   * @beta
+   */
+  export namespace appShareButton {
+    /**
+     * Property bag for the setVisibilityInfo
+     *
+     * @beta
+     */
+    export interface ShareInformation {
+      /**
+       * boolean flag to set show or hide app share button
+       */
+      isVisible: boolean;
+
+      /**
+       * optional string or null shareUrl, which will override shareUrl coming from Manifest
+       */
+      shareUrl?: string | null;
+    }
+    /**
+     * FrameContext of app manifest governs the default behavior of app share button
+     * example: if frameContext contains meeting stage then by default app share button is visible
```

Comment:
in current 60 day period, hide/show will be governed by manifest and api, example: if it is enabled in manifest and disabled by API then it will be a flash. After 60 days we will remove dependency from manifest and hide by default.

Reply:
I'd rather make this comment what the function will be like after the initial 60 days, otherwise you will need to come back and update this comment again in 2 months